### PR TITLE
Add translation to Danish

### PR DESCRIPTION
**Pull request by soegaard**

### CODE REVIEW

File: resources/lang/dk_DA.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+   "noRecordsFound":         "Ingen felter fundet",
+   "showingPage":            "Viser side {0} af {1}",
+   "show":                   "Vis ",
+   "search":                 "Søg",
+   "entries":                "indgange",
```
Comment (kevinanielsen) [inline resources/lang/dk_DA.json:6]: Translating "entries" to "indtastninger" would probably be more accurate.

```suggestion
   "entries":                "indtastninger",

```

File: resources/lang/dk_DA.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   "noRecordsFound":         "Ingen felter fundet",
+   "showingPage":            "Viser side {0} af {1}",
+   "show":                   "Vis ",
+   "search":                 "Søg",
+   "entries":                "indgange",
+   "columnName":             "Søjlenavn",
+   "insertANewColumnBefore": "Indsæt ny søjle før",
+   "insertANewColumnAfter":  "Indsæt ny søjle efter",
+   "deleteSelectedColumns":  "Slet valgte søjler",
+   "renameThisColumn":       "Omdøb denne søjle",
```
Comment (kevinanielsen) [inline resources/lang/dk_DA.json:11]: Translating "column" to "kolonne" would be more accurate.

```suggestion
   "columnName":             "Kolonnenavn",
   "insertANewColumnBefore": "Indsæt ny kolonne før",
   "insertANewColumnAfter":  "Indsæt ny kolonne efter",
   "deleteSelectedColumns":  "Slet valgte kolonner",
   "renameThisColumn":       "Omdøb denne kolonne",

```

File: resources/lang/dk_DA.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+   "editComments":           "Rediger kommentarer",
+   "addComments":            "Tilføj kommentarer",
+   "comments":               "Kommentarer",
+   "clearComments":          "Ryd kommentarer",
+   "copy":                   "Kopier...",
+   "paste":                  "Indsæt...",
+   "saveAs":                 "Gem som...",
+   "about":                  "Om",
+   "areYouSureToDeleteTheSelectedRows":                     "Sikker på at slette de valgte rækker?",
+   "areYouSureToDeleteTheSelectedColumns":                  "Sikker på at slette de valgte søjler?",
```
Comment (kevinanielsen) [inline resources/lang/dk_DA.json:26]: Same again.

```suggestion
   "areYouSureToDeleteTheSelectedColumns":                  "Sikker på at slette de valgte kolonner?",

```

**Review by kevinanielsen — COMMENTED** [review]
Left a few suggestions, otherwise it lgtm

**Review by hodeware — APPROVED** [review]

**Comment by hodeware** [comment]
thanks

**Pull request merged**